commons/connector/health/health_mergingExample1.avsc

```diff
@@ -0,0 +1,62 @@
+[
+  {
+    "namespace": "org.radarcns.active.healthkit",
+    "name": "HealthKitHeight",
+    "type": ["null", "record"],
+    "default": null,
+    "doc": "Height",
+    "fields": [
+      {
+        "name": "time",
+        "type": "double",
+        "doc": "Device timestamp in UTC (s)."
+      },
+      {
+        "name": "timeReceived",
+        "type": "double",
+        "doc": "Time that the data was received from the Apple Healthkit ."
+      },
+      { "name": "height", "type": "int", "doc": "Users's height, unit is m." }
```

Comment:
Thanks for updating this. I think the aggregation that we require would be for the actual fields. For example:
```{
   "namespace":"org.radarcns.active.healthkit",
   "name":"HealthKitBodyMeasurements",
   "type":[
      "null",
      "record"
   ],
   "default":null,
   "doc":"Health Kit body measurements..",
   "fields":[
      {
         "name":"time",
         "type":"double",
         "doc":"Device timestamp in UTC (s)."
      },
      {
         "name":"timeReceived",
         "type":"double",
         "doc":"Time that the data was received from the Apple Healthkit ."
      },
      {
         "name":"height",
         "type":"int",
         "doc":"Users's height, unit is m."
      },
      {
         "name":"weight",
         "type":"int",
         "doc":"Users's weight, unit is kg."
      },
      {
         "name":"bmi",
         "type":"int",
         "doc":"Users's bmi, unit is count."
      }
   ]
}.....